client-participation/gulpfile.js

```diff
@@ -235,7 +235,10 @@ gulp.task('embedJs', function() {
     'api/embed.js',
     'api/embedPreprod.js',
     'api/twitterAuthReturn.html',
-    ])
+  ])
+  .pipe(template({
+    polisHostName: polisConfig.SERVICE_HOSTNAME || 'pol.is',
```

Comment:
this would be inconsistent, but perhaps should be set in `process.env.SERVICE_HOSTNAME` instead of polisConfig, since then it would be simpler to set during CI. (Wouldn't need to set it to `localhost` in template polisConfig.)

```suggestion
    polisHostName: process.env.SERVICE_HOSTNAME || 'pol.is',
```